src/vs/workbench/contrib/preferences/browser/settingsWidgets.ts

```diff
@@ -713,7 +712,7 @@ export class ObjectSettingWidget extends AbstractListSettingWidget<IObjectDataIt
 
 	protected renderHeader() {
 		if (this.model.items.length > 0) {
-			const header = $('.setting-list-row-header');
+			const header = $('.setting-list-row-header', { 'aria-hidden': true });
```

Comment:
Why is it hidden?

Reply:
I looked into it, and it shouldn't be hidden.